src/middlewares/logger-middleware.ts

```diff
@@ -37,41 +44,14 @@ export class LoggerMiddleware implements PolarisMiddleware {
         context: any,
         info: GraphQLResolveInfo,
         result: string,
-    ) {
-        if (!root) {
-            const props: GraphqlLogProperties = this.initReqProperties(root, args, context, info);
-            this.polarisLogger.debug(
-                `Resolver of ${props.operationName} finished execution. Arguments received:
-             ${props.request && props.request.requestQuery && props.request.requestQuery.body}`,
-                props,
-            );
-        } else {
-            const props: GraphqlLogProperties = this.initReqProperties(root, args, context, info);
+    ): string {
+        const props: GraphqlLogProperties = LoggerMiddleware.buildProps(context);
+        if (root) {
```

Comment:
no need to check if we are/aren't in root because we're going to log this information through KOA